components/clp-package-utils/clp_package_utils/scripts/native/decompress.py

```
@@ -300,12 +300,21 @@ def main(argv):
+        type=str,
+        default=None,
+        help="The dataset that the archives belong to.",
+    )
     json_extraction_parser.add_argument(
         "--target-chunk-size", type=int, help="Target chunk size (B)."
     )
 
     parsed_args = args_parser.parse_args(argv[1:])
 
+    if parsed_args.dataset is not None:
+        validate_dataset(clp_config.database, parsed_args.dataset)
```

Flake8 (7.2.0)

[error] undefined name 'clp_config'

(F821)